index.js

```diff
@@ -275,9 +276,20 @@ module.exports = class Combobox {
 
   updateOpts() {
     this.cachedOpts.forEach((opt) => {
+      // configure display of options based on filtering
       opt.style.display = this.currentOpts.indexOf(opt) === -1 ? 'none' : '';
     });
 
+    // configure the innerHTML of each option based on what optionValues returns
+    this.currentOpts.forEach((actopt) => {
+      const newVal = this.config.optionValue(actopt);
+      if (this.config.multiselect) {
+        actopt.getElementsByClassName('label')[0].innerHTML = newVal;
```

Comment:
I don't think we need an if/else here...In either case we should be setting the innerHTML of the option to what `this.config.optionValue()` returns